engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -236,6 +223,7 @@ private void resetStats() {
     }
 
     private void preRenderUpdate(DefaultRenderingProcess.StereoRenderState stereoRenderState) {
+
         switch (stereoRenderState) {
             case MONO:
                 currentRenderingStage = WorldRenderingStage.DEFAULT;
```

Comment:
DefaultRenderingProcess.StereoRenderState and the WorldRenderingStage seems to be functionally equivalent enums. I suspect one of the two might be removable, hence removing this switch block. I'll work on this in a separate PR. I'm adding this comment here just in case somebody wants to provide feedback about it.